DOCS/xml-fields.md

```diff
@@ -0,0 +1,177 @@
+
+The following details are just a summary and recommendations on use of the fields. The formal schema definition is available in [`schema/ListedLicense.xsd`](../schema/ListedLicense.xsd).
+
+The "General structure" section below describes in plain language the way that license XML files are structured. Then, the "Template fields" section provides more specific details about the individual fields.
+
+Except where explicitly stated otherwise, assume that "license" means "license or license exception" in the following details.
+
+This document assumes that you have a basic familiarity with XML formatting, specifically the way that opening and closed tags work and are nested within one another, e.g.: `<license><text></text></license>`
+
+## General structure
+
+### Files and Filenames
+
+The XML template for each license is stored in a separate file in the [`src/` directory](../src) with the name `IDENTIFIER.xml`, where `IDENTIFIER` is the unique ID that the license will have on the License List.
+
+For license exceptions, the same pattern is used, with exceptions stored in the [`src/exceptions/` directory](../src/exceptions).
+
+A corresponding "test text" file, with a plain text version of the license or exception, should be added to the [`test/simpleTestForGenerator/` directory](../test/simpleTestForGenerator). Unlike the XML files, the same directory is used for the test files for licenses as well as exceptions. This text file will be used by the CI system to confirm that the XML template actually matches the "test text" for each license and exception.
+
+### Whitespace and formatting
+
+We aren't picky about tabs vs. spaces, number of spaces for indenting, etc.
+
+At some point maybe we'll pick a standard and update the existing files, but we haven't yet, so use whatever you like.
+
+### Beginning and end matter
+
+Within the .xml file, the first and last lines of the file should be the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+
+. . .
+
+</SPDXLicenseCollection>
+```
+
+### `<license>` or `<exception>` tag
+
+The main tag used to define the license or exception is, unsurprisingly, **`<license>`** or **`<exception>`**. All of the remaining content will be enclosed within a `<license></license>` or `<exception></exception>` pair of tags.
+
+There are two mandatory attributes for every `<license>` and `<exception>` tag:
+* `licenseId`: the unique SPDX Identifier for the license; should be identical to the filename
+* `name`: the longer "real name" of the license
+
+There are two additional attributes which are optional, but highly encouraged:
+* `isOsiApproved` (for licenses, not exceptions): either "true" or "false" based on whether this license has been approved by the [Open Source Initiative](https://opensource.org/licenses/alphabetical)
+* `listVersionAdded`: in which release version of the SPDX License List was the license first added, e.g. "3.19"
```

Comment:
Do we need to say what to put here for new submissions?

Reply:
Sure -- I'll update to make it clearer.